__mocks__/fs.js

```diff
@@ -19,12 +19,12 @@
 /**
  * External dependencies
  */
-// eslint-disable-next-line no-undef
+// eslint-disable-next-line no-undef  -- @todo
 const path = require('path');
```

Comment:
To fix this we should set `"env": { "node": true }` here:

https://github.com/GoogleForCreators/web-stories-wp/blob/700032264a04185f9ca2b5232e3fe61a8ca34313/.eslintrc#L315-L322